pubnub/test/unit/dx/push_test.dart

```diff
@@ -189,6 +189,33 @@ void main() {
               #environment: null
             }));
       });
+
+      test('listPushChannels delegate supported arguments', () async {
+        fakePubnub.returnWhen(
+            #listPushChannels,
+            Future.value(
+                ListPushChannelsResult.fromJson(['ch1', 'ch2', 'ch3'])));
+
+        await fakePubnub.listPushChannels('A332C23D', PushGateway.mpns,
+            start: 'ch2', count: 10);
+
+        var invocation = fakePubnub.invocations[0];
+
+        expect(invocation.isMethod, equals(true));
+        expect(invocation.memberName, equals(#listPushChannels));
+        expect(invocation.positionalArguments,
```

Comment:
You can also use [trailing commas](https://docs.flutter.dev/tools/formatting#using-trailing-commas) here, I leave it up to you